src/main/java/org/quiltmc/loader/impl/entrypoint/GamePatch.java

```diff
@@ -128,12 +129,18 @@ protected boolean isPublicInstance(int access) {
 		return ((access & 0x0F) == (Opcodes.ACC_PUBLIC | 0 /* non-static */));
 	}
 
-	public void process(QuiltLauncher launcher, Function<String, ClassReader> classSource, Consumer<ClassNode> classEmitter) {
+	public void process(QuiltLauncher launcher, String namespace, Function<String, ClassReader> classSource, Consumer<ClassNode> classEmitter) {
 		throw new AbstractMethodError(getClass() + " must override one of the 'process' methods!");
 	}
+	public void process(QuiltLauncher launcher, Function<String, ClassReader> classSource, Consumer<ClassNode> classEmitter) {
+		process(launcher, QuiltLoaderImpl.INSTANCE.tryGetGameProvider().getNamespace(), classSource, classEmitter);
```

Comment:
`tryGetGameProvider()` should be replaced with `getGameProvider()` (it has a better error message if it's not set).

Reply:
I have found a better way to do it that uses the [MappingConfiguration](https://github.com/QuiltMC/quilt-loader/pull/428/files#diff-0d54a473dc8b9723b082ad8a8a27309ee719e13c863cc58580ad0aaf479cf025) instead, and can get the namespace from the provided launcher varuable.